tests/components/husqvarna_automower/snapshots/test_sensor.ambr

```diff
@@ -210,7 +210,7 @@
     'last_changed': <ANY>,
     'last_reported': <ANY>,
     'last_updated': <ANY>,
-    'state': '2023-06-05T19:00:00+00:00',
+    'state': '2023-06-06T02:00:00+00:00',
```

Comment:
Why did the state change?

Reply:
Timezone change, which is what this PR was fixing